src/wazuh_modules/vulnerability_scanner/src/databaseFeedManager/databaseFeedManager.hpp

```diff
@@ -293,15 +236,41 @@ class TDatabaseFeedManager final : public Observer<nlohmann::json&>
         const auto updaterPolicy = TPolicyManager::instance().getUpdaterConfiguration();
         const std::string topicName = updaterPolicy.at("topicName");
 
-        m_feedDatabase = std::make_unique<TRocksDBWrapper>(DATABASE_PATH, false);
-
         m_translationsCache =
             std::make_unique<LRUCache<std::string, std::string>>(TPolicyManager::instance().getTranslationLRUSize());
 
-        // This initializes the vendor and os cpe maps and should be called before any scan or message processing.
-        if (reloadGlobalMapsStartup)
+        try
         {
-            reloadGlobalMaps();
+            m_feedDatabase = std::make_unique<TRocksDBWrapper>(DATABASE_PATH, false);
+
+            // This initializes the vendor and os cpe maps and should be called before any scan or message processing.
+            if (reloadGlobalMapsStartup)
+            {
+                // Try to load global maps from the database, if it fails we throw an exception to force the download of
+                // the complete feed.
+                reloadGlobalMaps();
+            }
+        }
+        catch (const std::exception&)
+        {
+            // If the database does not exist, we need to create it, but first we need to remove the directory,
+            // because probably it is corrupted.
```

Comment:
Nitpick: For better clarity
```suggestion
            // Create the database if it doesn't exist. We must remove any existing directory, as it may be corrupted.

```